lib/msal-common/src/authority/Authority.ts

```diff
@@ -47,13 +47,17 @@ export class Authority {
     public regionDiscoveryMetadata: RegionDiscoveryMetadata;
     // Proxy url string
     private proxyUrl: string;
+    // Logger object
+    public logger: Logger;
 
     constructor(
+        logger: Logger,
```

Comment:
I'm not super familiar with NodeJS concept of public API, so please do educate me, but... If `Authority` is part of public API, can we add a new non-null param to the ctor? Isn't it a breaking change?

I would recommend adding it as the last parameter an making is optional if this is indeed the case.

Reply:
That's right this would be breaking, however, this is not exposed to our end users and we've traditionally been ok with making breaking changes in msal-common as long as the changefile for msal-common signifies this contains a "major" change, preventing older versions of dependent libraries from picking up these changes.

So @Robbie-Microsoft please either make this optional at the end as Bogdan suggested, or make the change type "major"